vscode/src/extension.common.ts

```diff
@@ -67,5 +67,5 @@ export async function activate(
         console.error(error)
     }
 
-    return api
+    return new ExtensionApi(context.extensionMode)
```

Comment:
```suggestion
    return api
```
reverting this change fixed the integration test